source/content.css

```diff
@@ -566,6 +566,9 @@ a.tabnav-extra[href$='mastering-markdown/'] {
 .js-issue-row .mt-1.text-small.text-gray { /* Issue details line */
 	flex-basis: 100%;
 }
+.js-issue-row .h4 { /* Title */
+	max-width: 100%; /* #1518 */
```

Comment:
Unrelated, but didn't RG used to linkify issue mentions?

Reply:
Hmm, it does linkify the issue mention in the discussion inline diff, but not in the "Files changed" view.